datacapturing/src/test/java/de/cyface/datacapturing/backend/DataCapturingLocalTest.java

```diff
@@ -30,15 +30,15 @@
 import static org.mockito.Mockito.doNothing;
 import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
 
 /**
  * Tests the inner workings of the data capturing without any calls to the Android system. Uses fake data.
  *
  * @author Klemens Muthmann
- * @version 2.0.1
+ * @version 2.0.2
  * @since 2.0.0
  */
+@RunWith(RobolectricTestRunner.class)
```

Comment:
Roboelectric annotation habe ich hinzugefügt, da sonst ein "Log.d not-mocked" error kommt.